README.md

```diff
@@ -121,6 +121,7 @@ Attributes
  * `node["confluent"]["uid"]` : optional staticly assign a uid for above user (default=`unset` picks form system config)
  * `node["confluent"]["group"]` : The group that owns the Confluent package files and runs the services (default=`confluent`)
  * `node["confluent"]["gid"]` : optional staticly assign a gid for above group (default=`unset` picks form system config)
+ * `node["confluent"]["backup_templates"]` : toggle backups for template resources (default=`false`)
```

Comment:
Call out to enable backups you need to set this to the number of backups, https://docs.chef.io/resource_template.html